adhocracy-plus/templates/a4dashboard/includes/project_list_item.html

```diff
@@ -79,17 +79,29 @@ <h3 class="u-first-heading">
 </li>
 
 <div class="modal fade" id="project{{ project.pk }}DeleteModal" tabindex="-1" role="dialog" aria-labelledby="projectDeleteModalLabel" aria-hidden="true">
-    <div class="modal-dialog" role="document">
+    <div
+        class="modal-dialog modal-dialog-centered"
+        role="document"
+    >
         <div class="modal-content">
             <div class="modal-header">
-                <h2 class="modal-title" id="project{{ project.pk }}DeleteModal">{% blocktranslate with project_name=project.name %} Are you sure you want to delete the project '{{ project_name }}'? {% endblocktranslate %}</h2>
-                <button type="button" class="btn btn--close" data-bs-dismiss="modal" aria-label="Close"></button>
+                <h2 className="u-no-margin-bottom u-spacer-top-one-half">
```

Comment:
hm..yes, but that would affect mB as well right? also it would affect the Modal (and its siblings, e.g. ReportModal etc.) component, as I am using the same classes...

Reply:
it can be documented here without touching the other repos?!